browser/ui/views/frame/brave_browser_view.cc

```diff
@@ -263,6 +268,36 @@ BraveBrowserView::BraveBrowserView(std::unique_ptr<Browser> browser)
                             base::Unretained(this)));
   }
 
+  if (base::FeatureList::IsEnabled(tabs::features::kBraveSplitView) &&
+      browser_->is_type_normal()) {
+    SplitViewBrowserData::CreateForBrowser(browser_.get());
+
+    auto devtools_web_view =
+        std::make_unique<views::WebView>(browser_->profile());
+    devtools_web_view->SetVisible(false);
+    auto contents_web_view =
+        std::make_unique<ContentsWebView>(browser_->profile());
+    contents_web_view->SetVisible(false);
+
+    secondary_devtools_web_view_ =
+        contents_container_->AddChildView(std::move(devtools_web_view));
+    secondary_contents_web_view_ =
```

Comment:
Not sure why, but for me the secondary devtools panel is frozen, and not interactive at all. Changing the tab makes it interactive.

Reply:
I also managed to get into a weird state where the devtools panel was closed when I had the other tiled tab `open` and then showing when I focused the `actual` tab.

Reply:
Yeah, I didn't wire up dev tools web view yet, bc I'm still struggling to figure out how it works 😂 Let me follow up in a separate PR

Reply:
SGTM - FWIW, they're working heaps better since I rechecked out the branch.